src/SecTester.Bus/Internal/JsonHttpMethodEnumerationStringConverter.cs

```diff
@@ -1,55 +1,52 @@
 using System;
 using System.Collections.Generic;
+using System.Linq;
 using System.Net.Http;
+using System.Reflection;
+using System.Runtime.CompilerServices;
 using System.Text.Json;
 using System.Text.Json.Serialization;
 
 namespace SecTester.Bus.Internal;
 
 internal class JsonHttpMethodEnumerationStringConverter : JsonConverter<HttpMethod>
 {
-  private static readonly HttpMethod _patch = new("PATCH");
-  private static readonly HttpMethod _copy = new("COPY");
-  private static readonly HttpMethod _link = new("LINK");
-  private static readonly HttpMethod _unlink = new("UNLINK");
-  private static readonly HttpMethod _purge = new("PURGE");
-  private static readonly HttpMethod _lock = new("LOCK");
-  private static readonly HttpMethod _unlock = new("UNLOCK");
-  private static readonly HttpMethod _propfind = new("PROPFIND");
-  private static readonly HttpMethod _view = new("VIEW");
-  private static readonly HttpMethod _trace = new("TRACE");
+  private static readonly IEnumerable<HttpMethod> BaseMethods = typeof(HttpMethod)
+    .GetProperties(BindingFlags.Public | BindingFlags.Static | BindingFlags.DeclaredOnly)
+    .Where(x => x.PropertyType.IsAssignableFrom(typeof(HttpMethod)))
+    .Select(x => x.GetValue(null))
+    .Cast<HttpMethod>()
+    .ToList();
 
-  private static readonly Dictionary<string, HttpMethod> _map = new(StringComparer.InvariantCultureIgnoreCase)
+  private static readonly IEnumerable<HttpMethod> CustomMethods = new List<HttpMethod>
   {
-    { HttpMethod.Delete.ToString(), HttpMethod.Delete },
-    { HttpMethod.Get.ToString(), HttpMethod.Get },
-    { HttpMethod.Head.ToString(), HttpMethod.Head },
-    { HttpMethod.Options.ToString(), HttpMethod.Options },
-    { HttpMethod.Post.ToString(), HttpMethod.Post },
-    { HttpMethod.Put.ToString(), HttpMethod.Put },
-    { _patch.ToString(), _patch },
-    { _copy.ToString(), _copy },
-    { _link.ToString(), _link },
-    { _unlink.ToString(), _unlink },
-    { _purge.ToString(), _purge },
-    { _lock.ToString(), _lock },
-    { _unlock.ToString(), _unlock },
-    { _propfind.ToString(), _propfind },
-    { _view.ToString(), _view },
-    { _trace.ToString(), _trace },
+    new("PATCH"),
+    new("COPY"),
+    new("LINK"),
+    new("UNLINK"),
+    new("PURGE"),
+    new("LOCK"),
+    new("UNLOCK"),
+    new("PROPFIND"),
+    new("VIEW")
   };
 
+  private static readonly Dictionary<string, HttpMethod> Map = BaseMethods.Concat(CustomMethods).Distinct()
```

Comment:
Please use more obvious and clear name

```suggestion
  private static readonly Dictionary<string, HttpMethod> Methods = BaseMethods.Concat(CustomMethods).Distinct()
```